third_party/java/arcs/build_defs/internal/schemas.bzl

```diff
@@ -3,26 +3,18 @@
 Rules are re-exported in build_defs.bzl -- use those instead.
 """
 
-load("//devtools/build_cleaner/skylark:build_defs.bzl", "register_extension_info")
-load(":kotlin.bzl", "arcs_kt_library", "arcs_kt_plan")
-load(":manifest.bzl", "arcs_manifest")
 load(":tools.oss.bzl", "arcs_tool_schema2wasm")
-load(":util.bzl", "manifest_only", "replace_arcs_suffix")
+load(":util.bzl", "replace_arcs_suffix")
 
-def _run_schema2wasm(
-        name,
-        src,
-        deps,
-        out,
-        language_name,
-        language_flag,
-        wasm,
-        test_harness = False):
-    """Generates source code for the given .arcs schema file.
+def arcs_cc_schema(name, src, deps = [], out = None):
```

Comment:
It looks like you moved arcs_kt_shcema to kotlin.bzl. Do you want to move arcs_cc_schema to a file called cpp.bzl or cc.bzl too?

Reply:
I like that; updated.